console/config/schema.json

```diff
@@ -1273,6 +1273,10 @@
           "type": "map", 
           "description": "A map where each key is a user group name and the value is a map of the corresponding settings for that group."
         }, 
+        "hostname": {
+          "type": "string", 
+          "description": "The vhost name."
+        }, 
```

Comment:
In PR 255, I updated the policy attribute descriptions (including "id") to be more descriptive. For the hostname description, you could use something like this:

The hostname of the vhost. This vhost policy will be applied to any client connection that is directed to the hostname that you specify.

Reply:
will do.